.gitignore

```diff
@@ -212,4 +212,7 @@ Cargo.lock
 # Ignore code-workspaces
 *.code-workspace
 
+# Block crawler tool output
+**/tools/blocks-crawler/output/*
```

Comment:
Maybe better to create a separate .gitignore under crawler

Reply:
fixed